Release script is failing

The ESLint release script is failing with the following error:

```
/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/node_modules/ejs/lib/ejs.js:352
  throw err;
  ^

ReferenceError: ejs:30
    28| %>
    29| 
 >> 30| <% if (prereleaseMajorVersion) { %>
    31| ## Highlights
    32| 
    33| This is a summary of the major changes you need to know about for this version of ESLint.

prereleaseMajorVersion is not defined
    at eval (eval at compile (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/node_modules/ejs/lib/ejs.js:618:12), <anonymous>:57:8)
    at returnedFn (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/node_modules/ejs/lib/ejs.js:653:17)
    at Object.exports.render (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/node_modules/ejs/lib/ejs.js:416:37)
    at generateBlogPost (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/Makefile.js:180:24)
    at release (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/Makefile.js:293:5)
    at Function.target.ciRelease (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/Makefile.js:1146:5)
    at Object.global.target.(anonymous function) [as ciRelease] (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/node_modules/shelljs/make.js:36:40)
    at /var/lib/jenkins/workspace/Releases/ESLint Release/eslint/node_modules/shelljs/make.js:48:27
    at Array.forEach (<anonymous>)
    at Timeout._onTimeout (/var/lib/jenkins/workspace/Releases/ESLint Release/eslint/node_modules/shelljs/make.js:46:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! eslint@5.0.1 ci-release: `node Makefile.js ciRelease`
npm ERR! Exit status 1
```

--- a/Makefile.js
+++ b/Makefile.js
@@ -177,7 +177,10 @@ function generateBlogPost(releaseInfo) {
          * instead of getting matched with the `no-undef` rule followed by the string "ined".
          */
         .sort((ruleA, ruleB) => ruleB.length - ruleA.length);
-    const output = ejs.render(cat("./templates/blogpost.md.ejs"), Object.assign({ ruleList }, releaseInfo)),
+
+    const renderContext = Object.assign({ prereleaseMajorVersion: null, ruleList }, releaseInfo);
+
+    const output = ejs.render(cat("./templates/blogpost.md.ejs"), renderContext),
         now = new Date(),
         month = now.getMonth() + 1,
         day = now.getDate(),

